pdr_backend/lake/ohlcv_data_factory.py

```diff
@@ -85,8 +86,44 @@ def get_mergedohlcv_df(self) -> pl.DataFrame:
         asyncio.run(self._update_rawohlcv_files(fin_ut))
         logger.info("Update all rawohlcv files: done")
         rawohlcv_dfs = self._load_rawohlcv_files(fin_ut)
-        mergedohlcv_df = merge_rawohlcv_dfs(rawohlcv_dfs)
 
+        for feed in self.ss.feeds:
+            columns = [
+                "timestamp",
+                "open",
+                "high",
+                "low",
+                "close",
+                "volume",
+                "Cumulative Dollar Value",
+                "Cumulative Ticks",
+            ]
+            df = rawohlcv_dfs[str(feed.exchange)][str(feed.pair)]
+            for threshold_type in ["volume", "tick", "dollar"]:
+                if getattr(feed, f"{threshold_type}_threshold") is not None:
```

Comment:
this part can still be improved wrt. DRY principle, maybe add functions on the threshold itself.